ACadSharp.Tests/Entities/MTextValueReaderWriterTests.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using ACadSharp.Entities;
+using Xunit;
+using System.IO;
+using Xunit.Abstractions;
+using Newtonsoft.Json;
+
+namespace ACadSharp.Tests.Entities
+{
+	public class MTextValueReaderWriterTests
+	{
+		private readonly ITestOutputHelper _output;
+		private readonly Random _random;
+		private readonly JsonSerializerSettings _jsonSettings;
+
+		class MemoryConverter : JsonConverter
```

Comment:
Is it possible to use `System.Text.Json` instead of `Newtonsoft`?
Being this the test project I don't mind to leave it as it is but is just to have only one json serializer in the project.